smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/AddEventStreamDependency.java

```diff
@@ -44,6 +44,13 @@
 @SmithyInternalApi
 public final class AddEventStreamDependency implements TypeScriptIntegration {
 
+    @Override
+    public List<String> runAfter() {
+        return List.of(
+            AddBuiltinPlugins.class.getCanonicalName()
```

Comment:
Use the `name()` method. This is what the Directed codegen filters actually use when determining run order.

Reply:
No DI framework. Instantiate one to get the name?

Reply:
Yeah. That would be my recommendation. Then if the name ever changes you wont have things get out of sync.